discover/portal/articles/100-web-experience-management/02-building-a-site/03-creating-multiple-navigation-menus/00-creating-and-managing-navigation-menus-intro.markdown

```diff
@@ -13,10 +13,10 @@ direct users to content that is relevant to them.
 
 The default menu functions just like it always has: when you add a page, it's
 added to the menu. You can create sub-pages and build the page hierarchy any way
-you like. Go to *Navigation* &rarr; *Site Pages* to view the existing pages or
+you like. Go to *Navigation* &rarr; *Pages* to view the existing pages or
 create new pages.
```

Comment:
This paragraph needs work. If we say "just like it always has" we are assuming the reader knows "how it was". But that's not true for new readers. We should not assume all readers will be familiar with previous versions in order to understand the documentation.

My suggestion for this paragraph is the following (after word smithing, of course):
A site administrator can organize pages in a hierarchy while creating them. There are no limits to the depth of levels or the number of pages per page. The page hierarchy can be used for site navigation through the Navigation Menu widget (in fact it's the default for the widget). Go to *Navigation* &rarr; *Pages* to view the existing pages or
create new pages.
create new pages.